src/System.Private.CoreLib/shared/System/Enum.cs

```diff
@@ -266,7 +266,7 @@ private static string InternalFlagsFormat(RuntimeType enumType, EnumInfo enumInf
             return result;
         }
 
-        internal static ulong ToUInt64(object value)
+        internal static ulong ToUInt64(object? value)
```

Comment:
Does this expect `null` anywhere? I think this should be `object`

Reply:
Were no error for being nullable, updated